app/tests/services/test_otp_ticket_service.py

```
@@ -0,0 +1,51 @@
+    assert stored is None
+
+    # cleanup any ticket that was issued
+    for ticket in successes:
+        await redis_del(svc._ticket_key(ticket))
+
+
+@pytest.mark.asyncio
+async def test_wrong_otp_does_not_consume():
+    svc = OTPTicketService()
+    email = "wrong@example.com"
+    otp = (await svc.issue_otp(email)).otp
```

Ruff (0.14.14)

[error] Local variable `otp` is assigned to but never used

Remove assignment to unused variable `otp`

(F841)